integration-cli/docker_utils.go

```diff
@@ -503,9 +503,11 @@ func init() {
 	// intelligent decisions about how to configure themselves, and validate
 	// that the target platform is valid.
 	res, b, err := sockRequestRaw("GET", "/version", nil, "application/json")
-	defer b.Close()
-	if err != nil || res.StatusCode != http.StatusOK {
-		panic("Init failed to get version: " + err.Error() + " " + string(res.StatusCode))
+	if b != nil {
```

Comment:
Fair point. I went one better and just used `_` instead. No need for nil check or a close :)